x/plans/keeper/plan.go

```diff
@@ -20,8 +20,8 @@ func (k Keeper) AddPlan(ctx sdk.Context, planToAdd types.Plan, modify bool) erro
 		var planFromStore types.Plan
 		block, _, _, found := k.plansFS.FindEntryDetailed(ctx, planToAdd.GetIndex(), planToAdd.Block, &planFromStore)
 		if found {
-			if planFromStore.Price.Amount.LT(planToAdd.Price.Amount) {
-				return utils.LavaFormatError("failed modifying plan in planFS", fmt.Errorf("plan price cannot be increased"),
+			if !planFromStore.Price.Amount.Equal(planToAdd.Price.Amount) {
```

Comment:
why?
sorry if i dont remember

Reply:
We agreed that plan's price can't be modified.

But now that you're mentioning it, maybe we reverted that change? @omerlavanet

Reply:
cannot be increased, i dont remember the hard equal restriction